docker-compose.prod.yml

```diff
@@ -0,0 +1,79 @@
+# Production deployment configuration
+# Use with: docker-compose -f docker-compose.yml -f docker-compose.prod.yml up
+
+services:
+  postgres:
+    # Production postgres settings
+    deploy:
+      resources:
+        limits:
+          memory: 2G
```

Comment:
This is waaaaaay to optimistic.  I understand we can't just allocate 32GB for people who just want to try out the project but perhaps make this 4 or 8 GB